pages/common/clash.md

```diff
@@ -1,7 +1,7 @@
 # clash
 
 > A rule-based tunnel in Go.
-> More information: <https://github.com/Dreamacro/clash/wiki>.
+> More information: <https://web.archive.org/web/20231101052632/https://dreamacro.github.io/clash/>.
```

Comment:
In this case probably yes, since the command isn't well known and there are no proper maintained forks.

Generally for these cases we keep the pages for a while for people using old hardware or LTS Distros (Ubuntu, etc) so that they can check the usage for current version even if upstream becomes unavailable. Another thing that we do is check for actively maintained forks and suggest it instead in the modified page.

Reply:
> [fossabot/clash](https://github.com/fossabot/clash) looks like the "original" repository

Should I change the link to this one?

Reply:
This looks like a outdated repository (this one - 28 commits / [Dreamacro/clash](https://web.archive.org/web/20231101105407/https://github.com/Dreamacro/clash) - 928 commits)